.circleci/tests/golden/wiz-outpost-lite/container-registry-with-proxy.golden.yaml

```diff
@@ -92,6 +92,8 @@ spec:
         app.kubernetes.io/instance: release-test
         wiz.io/runner: "container-registry"
     spec:
+      securityContext:
```

Comment:
Lets make this back compat, if there's no sec ctx, the field would be omitted entirely